tokens/source/components/method.json

```diff
@@ -41,40 +41,94 @@
       },
       "text": {
         "connect": {
-          "comment": "Text color for the CONNECT method (purple.60).",
-          "value": "{color.alias.purple.60}"
+          "_": {
+            "comment": "Text color for the CONNECT method (purple.60).",
+            "value": "{color.alias.purple.60}"
+          },
+          "strong": {
```

Comment:
@two24studios I assumed the pattern we would want for these would be `kui-method-color-text-get-strong`, what they are used for is `:hover` state for icons in those method badges. Please let me know if we want to change names/values for these.